string current_floor
---
bool result
